### ISSUE TITLE
Improve logging

### ISSUE THREAD

**Issue by rbehjati** [project-oak/transparent-release#14]
Currently, we log errors from running the command (exec.Command) in a tempfile. 
These files are difficult to access on GitHub, in case, a CI step fails. 

Should we instead log everything to console? 

The problem with that is that the logs may get very messy and difficult to use.

**Comment by jul-sh** [comment]
I think this is a good idea. If messy logs an issue we can consider adding a "verbose" flag?

**Comment by rbehjati** [comment]
Yes. I think a `verbose` flag would be a good idea :) Not sure how to implement it nicely though.
